aloha/robot_utils.py

```diff
@@ -209,21 +137,26 @@ def sleep_arms(
     """
     if home_first:
         move_arms(
-            bot_list,
-            [[0.0, -0.96, 1.16, 0.0, -0.3, 0.0]] * len(bot_list),
+            bot_list=bot_list,
+            DT=DT,
+            target_pose_list=[
+                [0.0, -0.96, 1.16, 0.0, -0.3, 0.0]] * len(bot_list),
             moving_time=moving_time
         )
     move_arms(
-        bot_list,
-        [bot.arm.group_info.joint_sleep_positions for bot in bot_list],
+        bot_list=bot_list,
+        target_pose_list=[
+            bot.arm.group_info.joint_sleep_positions for bot in bot_list],
         moving_time=moving_time,
+        DT=DT
     )
 
 
 def move_grippers(
     bot_list: Sequence[InterbotixManipulatorXS],
     target_pose_list: Sequence[float],
     moving_time: float,
+    DT
```

Comment:
Make sure to add a comma even to the last parameter so that changes to the signature look clean later

```suggestion
    DT,
```